boudicca.events/eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/LinzTermineCollector.kt

```diff
@@ -137,7 +137,7 @@ class LinzTermineCollector : EventCollector {
 
         var date = LocalDate.now(ZoneId.of("Europe/Vienna")).atStartOfDay()
         val links = mutableListOf<String>()
-        for (i in 1..(4 * 6)) {
+        for (ignored in 1..(4 * 6)) {
```

Comment:
ignored?

Reply:
i cannot use "_" as a name for a variable in kotlin in a for loop, so i have to use "ignored" to signify that i don't care about it. otherwise i have an unused variable